docker/Dockerfile.1-13-cpu

```diff
@@ -0,0 +1,27 @@
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION} as tfs
+FROM ubuntu:18.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
```

Comment:
Just FYI, AWS TF team produces optimized binary which we will consume in future.

Reply:
for tensorflow serving or just tensorflow?

Reply:
Both